el-patch.el

```diff
@@ -222,6 +226,13 @@ This function lives halfway between `copy-sequence' and
       (cons (car tree) (el-patch--copy-semitree (cdr tree)))
     tree))
 
+(defun el-patch--advice-name (name variant)
```

Comment:
This can be confusing since everywhere else, `variant` is already `(advice . variant-name)` except here where it's just `variant-name`.